includes/get-involved.md

```diff
@@ -1,6 +1,6 @@
 If you're interested in contributing to or participating in the direction of KEDA, you can join our community meetings.
 
-* Meeting time: Bi-weekly Thurs 17:00 UTC (does follow US daylight savings). (Subscribe to [Google Agenda](https://calendar.google.com/calendar?cid=bjE0bjJtNWM0MHVmam1ob2ExcTgwdXVkOThAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) | [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=Seattle&))
+* Meeting time: Bi-weekly Thurs 16:00 UTC (does follow US daylight savings). (Subscribe to [Google Agenda](https://calendar.google.com/calendar?cid=bjE0bjJtNWM0MHVmam1ob2ExcTgwdXVkOThAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) | [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=Seattle&))
```

Comment:
We should modify the timezoneconverter link as well to https://www.thetimezoneconverter.com/?t=16%3A00&tz=UTC%20(Coordinated%20Universal%20Time)&